.github/workflows/hqs-ci-test-pure-python.yml

```diff
@@ -0,0 +1,28 @@
+name: HQS CI tests for pure python repos/folders
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  linting:
+    uses: HQSquantumsimulations/reusable_workflows/.github/workflows/reusable_linting_pure_python.yml@main
+    with: 
+      python_folder: "qoqo_qiskit"
+      linting_folder: "qoqo_qiskit/qoqo_qiskit"
+      # Run tests also on windows runners
+      windows: false
+      # Run tests also on macos runners
+      macos: true
+
+  unittests:
+    uses: HQSquantumsimulations/reusable_workflows/.github/workflows/reusable_tests_pure_python.yml@main
+    with:
+      python_folder: "qoqo_qiskit"
+      # Test code coverage of rust core is over 80 percent
+      test_code_coverage: true
+      # Run tests also on windows runners
+      windows: false
+      # Run tests also on macos runners
+      macos: true
```

Comment:
Are these boolean flags correct?

Reply:
no and I have just opened a bug report because I took it from the `.github` repo